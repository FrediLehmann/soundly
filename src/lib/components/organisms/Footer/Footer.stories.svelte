<script>
  import { Meta, Story } from '@storybook/addon-svelte-csf';

  import Footer from './Footer.svelte';
</script>

<Meta title="Organisms/Footer" />

<Story name="Footer">
  <Footer />
</Story>
