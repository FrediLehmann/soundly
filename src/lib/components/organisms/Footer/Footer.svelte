<footer>
  <div>Footer</div>
</footer>

<style>
  footer {
    position: absolute;
    bottom: 0;

    width: 100vw;

    border-top: 1px solid hsla(var(--neutral-300));
    background-color: hsla(var(--neutral-100));

    padding-block: 1.25rem;
  }

  footer > div {
    margin-inline: auto;
    max-width: 72rem;
  }
</style>
