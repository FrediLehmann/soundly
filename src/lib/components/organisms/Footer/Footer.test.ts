/**
 * @jest-environment jsdom
 */

import '@testing-library/jest-dom';
import { render } from '@testing-library/svelte';

import Footer from './Footer.svelte';

describe('<Footer /> component', () => {
  test('renders correctly', () => {
    const { component } = render(Footer);
    expect(component).toMatchSnapshot();
  });
});
