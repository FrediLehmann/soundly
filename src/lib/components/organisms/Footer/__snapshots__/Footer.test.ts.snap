// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> component renders correctly 1`] = `
Footer {
  "$$": Object {
    "after_update": Array [],
    "before_update": Array [],
    "bound": Object {},
    "callbacks": Object {},
    "context": Map {},
    "ctx": Array [],
    "dirty": Array [
      -1,
    ],
    "fragment": Object {
      "c": [Function],
      "d": [Function],
      "i": [Function],
      "l": [Function],
      "m": [Function],
      "o": [Function],
      "p": [Function],
    },
    "not_equal": [Function],
    "on_destroy": Array [
      [Function],
    ],
    "on_disconnect": Array [],
    "on_mount": Array [],
    "props": Object {},
    "root": <div>
      <footer
        class="svelte-7xqz41"
      >
        <div
          class="svelte-7xqz41"
        >
          Footer
        </div>
      </footer>
    </div>,
    "skip_bound": false,
    "update": [Function],
  },
}
`;
